src/atomate2/common/flows/phonons.py

```diff
@@ -354,3 +365,15 @@ def make(
 
         # create a flow including all jobs for a phonon computation
         return Flow(jobs, phonon_collect.output)
+
+    @property
+    @abstractmethod
```

Comment:
`@abstractmethod` was added here and it looks correct to me. But it seems missing in vasp/aims/forcefields.